tests/test_entry.py

```diff
@@ -0,0 +1,43 @@
+"""
+ Pylibui test suite.
+
+"""
+
+from pylibui.controls import Entry, PasswordEntry, SearchEntry
+from tests.utils import WindowTestCase
+
+
+class EntryTest(WindowTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.entry = Entry()
+        self.password_entry = PasswordEntry()
+        self.search_entry = SearchEntry()
```

Comment:
Did you used `password_entry` and `search_entry` in this file, am I overlooking them? If you didn't use I think we can remove them because they all share the same functions, am I wrong?